### ISSUE TITLE
Needs better demos/examples

### ISSUE THREAD

**Issue by khawtf** [kriasoft/react-static-boilerplate#206]
I really like the boilerplate but the demo/example application is really lacking.
Especially with the custom routing. I see allot of issues referring to routing issues because the concept is not really clear.

I read the doc that came with it. But to make this boilerplate more "standard" it should have a bit more complex demo/example application for devs to dive into.

Just my 2 cents.

**Comment by DaveyEdwards** [comment]
I agree docs are not quite like the react-starter-kit yet.  I am in the process of transferring my project over and would like to help with docs while doing so.  

If we could get a list going of things that we want in the docs that would be very helpful, post them up in this thread.
